desktop/core/src/desktop/js/parse/sql/generic/genericAutocompleteParser.Locations.test.js

```diff
@@ -46,6 +46,13 @@ describe('genericAutocompleteParser.js locations', () => {
     });
   };
 
+  it.skip('useful for testing', () => {
+    assertLocations({
+      beforeCursor: 'select * from a join b on a.a = b.b::date order by ',
+      expectedLocations: []
+    });
+  });
```

Comment:
Changing the `skip` to `only` is a really easy way to test out changes to the parser while you're working on things so going to leave this in as a utility if it isn't too irksome